src/lib/y2network/interface_config_builder.rb

```diff
@@ -83,6 +83,19 @@ def driver=(value)
       Yast::LanItems.setDriver(value)
     end
 
+    def driver_options
+      target_driver = driver
+      if target_driver.empty?
+        target_driver = Yast::Ops.get_string(Yast::LanItems.getCurrentItem, ["hwinfo", "module"], "")
+      end
```

Comment:
we have Hwinfo class so we can relax dependency on LanItems here

```
Hwinfo.new(name).module
````
(module method is currently missing in the Hwinfo class but it is very easy to add)